code/datums/status_effects/status_effect.dm

```diff
@@ -151,28 +151,45 @@
 	id = "stacking_base"
 	duration = -1 //removed under specific conditions
 	alert_type = null
-	var/stacks = 0 //how many stacks are accumulated, also is # of stacks that target will have when first applied
-	var/delay_before_decay //deciseconds until ticks start occuring, which removes stacks (first stack will be removed at this time plus tick_interval)
-	tick_interval = 10 //deciseconds between decays once decay starts
-	var/stack_decay = 1 //how many stacks are lost per tick (decay trigger)
-	var/stack_threshold //special effects trigger when stacks reach this amount
-	var/max_stacks //stacks cannot exceed this amount
-	var/consumed_on_threshold = TRUE //if status should be removed once threshold is crossed
-	var/threshold_crossed = FALSE //set to true once the threshold is crossed, false once it falls back below
+
+	/// How many stacks are accumulated, also is # of stacks that target will have when first applied
+	var/stacks = 0
+	/// Deciseconds until ticks start occuring, which removes stacks (first stack will be removed at this time plus tick_interval)
+	var/delay_before_decay
+	/// Deciseconds between decays once decay starts
+	tick_interval = 10
```

Comment:
New vars must be declared after old vars

Reply:
You sure you want me to put that below the image vars? I believe it looks nice just the way it is. Can put it down there if you prefer it, though.